libs/http-ext/src/lib/matchers/match-origin/match-origin.spec.ts

```diff
@@ -4,21 +4,37 @@ import { matchOrigin } from './match-origin';
 describe.each([
   /* Using a string */
   ['https://test.com', 'https://test.com', true],
+  // @todo ['https://test.com/test', 'https://test.com', true],
   ['https://test.com', 'https://angular.io', false],
+  // @todo ['https://test.com.test.com', 'https://test.com', false],
```

Comment:
We've got to make sure we are really testing the origin and not just using a `startsWith`.